chunky/src/java/se/llbit/chunky/ui/render/tabs/WaterTab.java

```diff
@@ -132,21 +132,23 @@ public Node getTabContent() {
   @Override
   public void initialize(URL location, ResourceBundle resources) {
     waterVisibility.setName("Water visibility");
-    waterVisibility.setTooltip("Visibility depth under water.");
+    waterVisibility.setTooltip("Distance of visibility past the surface of the water.");
     waterVisibility.setRange(0, 50);
     waterVisibility.clampMin();
     waterVisibility.onValueChange(value -> scene.setWaterVisibility(value));
 
     waterOpacity.setName("Water opacity");
-    waterOpacity.setTooltip("Sets how opaque the water surface appears.");
+    waterOpacity.setTooltip("Opacity of the surface of the water.");
     waterOpacity.setRange(0, 1);
     waterOpacity.clampBoth();
     waterOpacity.onValueChange(value -> scene.setWaterOpacity(value));
 
+    stillWater.setTooltip(new Tooltip("Disable the ripple effect of the water."));
```

Comment:
More commonly known as "waves" :smile: 

```suggestion
    stillWater.setTooltip(new Tooltip("Disable the waves on top of the water."));
```